src/Symfony/Component/AssetMapper/Compiler/JavaScriptImportPathCompiler.php

```diff
@@ -27,59 +28,65 @@ final class JavaScriptImportPathCompiler implements AssetCompilerInterface
 {
     use AssetCompilerPathResolverTrait;
 
-    // https://regex101.com/r/VFdR4H/1
-    private const IMPORT_PATTERN = '/(?:import\s+(?:(?:\*\s+as\s+\w+|[\w\s{},*]+)\s+from\s+)?|\bimport\()\s*[\'"`](\.\/[^\'"`]+|(\.\.\/)+[^\'"`]+)[\'"`]\s*[;\)]?/m';
+    // https://regex101.com/r/5Q38tj/1
+    private const IMPORT_PATTERN = '/(?:import\s+(?:(?:\*\s+as\s+\w+|[\w\s{},*]+)\s+from\s+)?|\bimport\()\s*[\'"`](\.\/[^\'"`]+|(\.\.\/)?[^\'"`]+)[\'"`]\s*[;\)]?/m';
```

Comment:
why this change?

Reply:
We're looking for `import` statements inside JS files. Before this PR, we did this for only one reason: to find relative imports (`import './foo.js'`) so we could automatically add those to the importmap.

Now, there is one other purpose: to know which files we should preload (i.e. if `app.js` imports `foo.js`, and `app.js` is the entrypoint, then preload `foo.js`). This preloading applies now to ALL imports - relative or "bare" (i.e. `import 'bootstrap'`). This regex change allows us to match both relative `import './foo.js'` and bare `import 'foo'` imports.

Reply:
this should then be a `*` instead of a `?`?

Reply:
Both seem to work, but I see your point. Changed to `*`